File: Práctica 1/C++/TurRelevante.cc
//DNI 23900715N HERNANDEZ GOMEZ, JESSICA
#include "Coleccion.h"

using namespace std;

int main(int argc, char const *argv[]){
    Coleccion colec = Coleccion();

    if(argc == 2){
        colec.lectura(argv[1]);

        for(unsigned int x = 0; x < colec.getLocalidades().size(); x++){
				cout << colec.getLocalidades()[x].getNombre() << " (" << colec.getLocalidades()[x].getCoor().getFila() << "," << colec.getLocalidades()[x].getCoor().getColumna() << "): ";
				cout << colec.getLocalidades()[x].getInfo().getMasFrecuente();
				
				if(colec.getLocalidades()[x].getInfo().getTop() != ""){
					cout << " - " << colec.getLocalidades()[x].getInfo().getTop();
				}
				cout << endl;
			}
    }
}

File: Práctica 1/Java/TurRelevante.java

//23900715N HERNANDEZ GOMEZ, JESSICA
import java.util.*;

public class TurRelevante {
	public static void main(String[] args) {
		Coleccion colec = new Coleccion();

		if (args.length == 1) {
			if (args[0] != null) {
				colec.lectura(args[0]);

				for (int x = 0; x < colec.getLocalidades().size(); x++) {
					System.out.print(colec.getLocalidades().get(x).getNombre() + " (" + colec.getLocalidades().get(x).getCoor().getFila() + "," + colec.getLocalidades().get(x).getCoor().getColumna() + "): ");
					System.out.print(colec.getLocalidades().get(x).getInfo().getMasFrecuente());

					if (colec.getLocalidades().get(x).getInfo().getTop().equals("") == false) {
						System.out.print(" - " + colec.getLocalidades().get(x).getInfo().getTop());
					}
					System.out.println();
				}
			}
		}
	}
}
